cmd/snapd-apparmor/main.go

```diff
@@ -68,19 +69,32 @@ import (
 // any loss of functionality. This is an unsupported configuration that cannot
 // be properly handled by this function.
 func isContainerWithInternalPolicy() bool {
+	var securityFSPath = filepath.Join(dirs.GlobalRootDir, "/sys/kernel/security")
+
 	if release.OnWSL {
-		// WSL-1 is an emulated Windows layer that has no support for AppArmor.
-		// WSL-2 is a virtualised environment with the Linux kernel as
-		// distributed by Microsoft.  In the future, if Microsoft enables
-		// AppArmor in their kernel configuration and adjusts their special
-		// init process, which launches individual distributions in
-		// quasi-containers, to initialize apparmor nesting, we might re-enable
-		// this and treat WSL-2 like a LXD/Incus container, with nested
-		// security.
+		if release.WSLVersion == 2 && osutil.IsDirectory(securityFSPath) {
+			// WSL-1 is an emulated Windows layer that has no
```

Comment:
Should this comment about WSL-1 and WSL-2 live above the `if release.WSLVersion == 2 ...` check? I think this would make more sense.

Reply:
Done